plugins/services/src/js/components/HealthBar.js

```diff
@@ -1,121 +0,0 @@
-import classNames from "classnames";
-import React from "react";
-import { Tooltip } from "reactjs-components";
-
-import ProgressBar from "#SRC/js/components/ProgressBar";
-import StringUtil from "#SRC/js/utils/StringUtil";
-
-import HealthBarStates from "../constants/HealthBarStates";
-
-class HealthBar extends React.Component {
```

Comment:
@Poltergeist please take a look at this commit https://github.com/dcos/dcos-ui/pull/2475/commits/4b87367e0f9f4f555c795941fab4b941c1f309b6

This was renamed to `ServiceStatusProgressBar` and was refactored so because git sees this as deleting and adding that's what we get for both files

`HealthBar.js` deleted
`ServiceStatusProgressBar.js` added

Reply:
Okay that makes sense now. I didn't drew the connection, that it actually was a rename.